system/services/admin/AuditService.cfc

```diff
@@ -32,6 +35,47 @@ component output="false" singleton=true {
 		} );
 	}
 
+	public query function getAuditTrailLog(
+		  numeric startRow = 1
+		, numeric maxRows  = 0
+	) {
+		var records = _getDao().selectData(
+			  selectFields = [ "audit_log.id", "audit_log.type", "audit_log.datecreated", "audit_log.action", "audit_log.detail" , "security_user.email_address", "security_user.known_as"  ]
+			, orderby      = "audit_log.datecreated desc"
+			, startRow     = arguments.startRow
+			, maxRows      = arguments.maxRows
+		);
+
+		return records;
+	}
+
+	/**
+	 * Returns the auditTrail record matching the given ID
+	 *
+	 * @autodoc
+	 * @auditLogId.hint The ID of the audit log you wish to get
+	 *
+	 */
+	public query function getAuditLog( required string auditLogId ) {
+		return $getPresideObject( "audit_log" ).selectData(
+			filter = { id=auditLogId }
+		);
+	}
+
+	public any function renderLogMessage(  required struct log ) {
+
+		var viewletEvent = "renderers.auditLog.auditLogEntry." & arguments.log.action;
+		if ( $getColdbox().viewletExists( viewletEvent ) ) {
+			return $getColdbox().renderViewlet(
+				  event = viewletEvent
+				, args  = arguments.log);
```

Comment:
closing parenthesis belongs on a new line and should line with the beginning of the first line (return). So:

```
			return $getColdbox().renderViewlet(
				  event = viewletEvent
				, args  = arguments.log
			);
```

Same for function call below.